### ISSUE TITLE
ALL: aria-labelledby is unsupported when used on the header element

### ISSUE THREAD

**Issue by imcbride** [middlebury/midd-frontend#540]
https://my2.siteimprove.com/Inspector/7333/A11Y/Page?pageId=64768883914&impmd=ERAQQMIEOUHRRKBGATMU&decision=dismissed+falsePositive+none#/sia-r18/failed/NHJVUlViTTc=;ZTNlMzJlOTA=

https://github.com/middlebury/midd-frontend/blob/master/src/templates/partials/page-header.twig#L3

**Comment by poojagunturu96** [comment]
It seems like this is true. According to [this article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header#accessibility), the `<header>` element defines a [banner](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/banner_role) landmark when its context is the [`<body>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body) element. A banner role does not have any associated [associated ARIA roles, states, and properties](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/banner_role#associated_aria_roles_states_and_properties), which means that a header tag cannot have an `aria-labelledby` attribute on it. 
Siteimprove seems to be correctly reporting this issue, so it probably makes sense to remove that attribute from `header`s?

**Comment by imcbride** [comment]
I tested this out by removing the `aria-labelledby` attribute from the `header` element in `page-header.twig` and it passed the static analysis checks in the WAVE and SiteImprove plugins, so I agree this is probably the direction we should take on this issue.
